change/@uifabric-charting-2020-07-02-14-22-06-user-v-gorraj-Task_4265281.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
>patch [](start = 11, length = 5)

should this be patch? You are adding a new props.

Reply:
I have changed this to minior as per fabric guidelines